### PR TITLE
Day 4 assingment

### PR DESCRIPTION
**Pull request by uditpratap-successive**

### CODE REVIEW

File: day.4/Ques1.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//Write a program to remove duplicate characters from string ("Hello Yellow" => "Helo Yw")
```
Comment (saurabh-successive92) [inline day.4/Ques1.js:1]: Please check this.It is not right solution

File: day.4/Ques3.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    let freq={};
+
+    for(let i=0;i<s.length;i++)
+    {
+        if(freq[s[i]])
+        freq[s[i]]+=1;
+        else
+        freq[s[i]]=1;
+    }
+    //console.log(freq);
```
Comment (saurabh-successive92) [inline day.4/Ques3.js:12]: remove this statement

File: day.4/Ques2.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const reverser = (string) => {
+    let splitstring=string.split(" ");
+    for(let i=0;i<splitstring.length;i++)
```
Comment (saurabh-successive92) [inline day.4/Ques2.js:3]: Please do it without loop

File: day.4/ques11.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+//Write a program to sort an array (bubble sort)
```
Comment (saurabh-successive92) [inline day.4/ques11.js:1]: Please check this logic with different use cases

File: day.4/Ques3.js (hunk 2 above)
Comment (uditpratap-successive) [reply day.4/Ques3.js]: Done

File: day.4/Ques2.js (hunk 3 above)
Comment (uditpratap-successive) [reply day.4/Ques2.js]: I have done it without using for loop , instead for loop I used map to iterate over and apply the function.

File: day.4/ques11.js (hunk 4 above)
Comment (uditpratap-successive) [reply day.4/ques11.js:1]: I have tried with different use cases

**Review by saurabh-successive92 — APPROVED** [review]

**Pull request merged**